### ISSUE TITLE
Use another genome not in your references

### ISSUE THREAD

**Issue by El-Castor** [Zhang-lab/ATAC-seq_QC_analysis#3]
Hi, 

I want to used your software to do a QC on my ATACseq data performed on cucumis melo species. I'm wondering if it's possible and how I can integrate my genome on your image ? 

Thanks in advances

**Comment by ShaopengLiu1** [comment]
Hello,

Yes, it's possible (but a little trivial)

1. Please see here for what resource files you will need: `https://github.com/Zhang-lab/ATAC-seq_QC_analysis/blob/master/pipe_code/qc_source.sh`, briefly you need: BWA-ref, chrom_size, black_list (make an empty file if you don't have this), promoter file, coding promoter file and genome size parameter for macs2 (`https://github.com/taoliu/MACS`)

2. create all the files locally, then bind the folder path into Singularity `-B <path-to-parent-folder-of-ref-file>:/atac_seq/Resource/Genome`

3. modify the `qc_source.sh` file to add your local reference files (you may need to create a new image) to store the updated file. 

Or if you can give me your script for generating all of those files above, I can add to the reference genome from my side, which would be more friendly for you to use

**Comment by El-Castor** [comment]
Hello,

Thanks for your response, I' ve a question :
what do you mean for promoter file and coding promoter, I mean what is the
difference between the two file. coding_promoter are the promoter of the
only coding gene ? and the promoter_file it's all the promoter of the
genome features ?

Thanks

Le mar. 15 oct. 2019 à 02:20, Shaopeng Liu <notifications@github.com> a
écrit :

**Comment by ShaopengLiu1** [comment]
Yes, coding promoter is the promoter regions of only coding gene, for ATAC-seq (open chromatin) coding genes, they usually have stronger signals.

Promoter file is for all gene features (including but more than coding gene). The relatively lower enrichment signal of non-coding genes will lead to a lower enrichment level.

**Comment by ShaopengLiu1** [comment]
The image build files are here if you need: "https://github.com/Zhang-lab/ATAC-seq_QC_analysis/tree/master/image_build_file"

I am trying to add a personlized input file as a parameter, should update soon, this will make your process easier.
